setup.py

```diff
@@ -24,9 +24,8 @@ def find_packages(path='.', prefix=""):
 requirements = [
     'Click>=6.0',
```

Comment:
can we rev to click 7.0 while we're here?

Reply:
This is a >= requirement so new installs will get the latest final release. Is there something we know is wrong with 6.x?